### ISSUE TITLE
Layout of the admin main page

### ISSUE THREAD

**Issue by tienne-B** [TabbycatDebate/tabbycat-frontend#7]
The hardest page must be the first page a user will see when they select a tournament!

In order from top to bottom, how about:
* Full-width link button prompting set-up actions (add participants if there are none) if applicable
* Card with the current round as title ([Select Button](https://primevue.org/selectbutton/) if many concurrent rounds), and [Stepper](https://primevue.org/steps/) to indicate round progression (with links)
* _Split_
  * Recent actions
  * Ballot graph?

Don't think the "Recent Results" card gave much value in the old interface.

**Comment by linh-trant** [comment]
About the _split_, two things:
- Is it feasible to have database actions in recent actions as well? I don't think we have this in our current version. 
- Showing feedback response rate or ballot status is also good for the second side.

**Comment by valpang22** [comment]
If possible for recent actions, can actions within config be displayed in the change log? Eg. [user] edited Draw Rules in Configurations.
Likely less of an issue with permissions being more widely adapted, but would still be nice if this can be recorded.

**Comment by valpang22** [comment]
> * Showing feedback response rate or ballot status is also good for the second side.

Agreed! Feedback response rate data is more widely referenced in tournaments, would be nice to have a graph to display the data cleanly
